code/game/objects/structures/crates_lockers/closets/secure/operations.dm

```diff
@@ -47,6 +49,12 @@
 	new /obj/item/modular_computer/handheld/preset/supply/cargo_delivery(src)
 	new /obj/item/export_scanner(src)
 	new /obj/item/device/flashlight/marshallingwand(src)
+	new /obj/item/clothing/head/deckhelmet(src)
+	new /obj/item/clothing/head/deckhelmet/green(src)
```

Comment:
Get rid of everything else other than the default helmet if you're going to put them in the loadout anyway.

Reply:
That's fair enough, I'd just been thinking of a way to have the other kinds available outside of the loadout, too